### ISSUE TITLE
TradingView chart screenshot URL fix

### ISSUE THREAD

**Issue by alexpunct** [timelyart/Kairos#28]
It seems that the TradingView chart screenshot URL is not a html page that includes the image (example: https://www.tradingview.com/x/yThC9lEo/). We should update Kairos to find the image in that html page and send that instead (https://s3.tradingview.com/snapshots/k/kKPjceQ6.png). 

Here's a regex to find the right image in the html page:
```
(https:\/\/s3\.tradingview\.com\/snapshots\/[a-z]+\/[a-zA-Z0-9]+\.png)
```

**Comment by timelyart** [comment]
Thanks for sharing your solution.

**Comment by timelyart** [comment]
When TV rearranged how screenshot where shared there was a period of time where you could only share the image through Twitter. However, it seems that it is now possible to just open screenshot in a new tab and extract the url to image from there.

Your original point is still valid though. My guess assumption is that they share a url to webpage with the image instead of the image itself to save on backend resources.

**Comment by timelyart** [comment]
Fixed with [release v2.53](https://github.com/timelyart/Kairos/releases/tag/v2.53).
